src/pages/WorkspaceSwitcherPage/index.tsx

```diff
@@ -175,6 +174,22 @@ function WorkspaceSwitcherPage() {
                 title={translate('workspace.switcher.headerTitle')}
                 onBackButtonPress={Navigation.goBack}
             />
+            <View style={[styles.mh4, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, styles.mb1]}>
+                <Text
+                    style={styles.label}
+                    color={theme.textSupporting}
+                >
+                    {translate('workspace.switcher.everythingSection')}
+                </Text>
+            </View>
+            <UserListItem
+                item={defaultPolicy}
+                isFocused={activeWorkspaceID === undefined}
```

Comment:
this is causing https://github.com/Expensify/App/pull/41158#issuecomment-2106184835
not sure why